src/client/src/utils/validateName.ts

```diff
@@ -1,5 +1,7 @@
 import { defineMessages, FormattedMessage } from "react-intl";
 
+const PROJECT_NAME_CHARACTER_LIMIT = 50;
```

Comment:
just curious, why 50?

Reply:
Our thought is that developers wouldn't put in such a long project name, and Visual Studio also limit the project name to 50 characters.

Reply:
We also based the project name input off of the Windows Template Studio char limit for project name!

Reply:
Very small nit that comes from this though, can we add this to the constants file, and use the constant here and ProjectNameOutput?